apps/expo/src/app/(dashboard)/index.tsx

```diff
@@ -6,35 +6,59 @@ import { Route, router } from 'expo-router'
 
 import { FontAwesome5 } from '@expo/vector-icons'
 
+import RotatingBarbellIcon from '~/components/notif/RotatingBarbellIcon'
+import PostDashboard from '~/components/post/PostDashboard'
 import Button from '~/components/ui/button/button'
 import BarbellTitle from '~/components/ui/nav-bar/BarbellTitle'
 import NavBar from '~/components/ui/nav-bar/NavBar'
+import { useGlobalContext } from '~/context/global-context'
+import ActivityFeed from '../activity/feed'
 
-// FIXME: Dashboard start here
 const Dashboard = () => {
+  const { userData } = useGlobalContext()
+
   return (
     <SafeAreaView style={{ backgroundColor: '#1E1E1E', flex: 1 }}>
-      <NavBar
-        left={<View />}
-        center={<BarbellTitle />}
-        showDivider={false}
-        right={
-          <TouchableOpacity
-            testID='message-button'
-            onPress={() => router.push('notif/' as Route<string>)}
-          >
-            <FontAwesome5 name='inbox' size={24} color='#CACACA' />
-          </TouchableOpacity>
-        }
-      />
+      {userData ? (
+        <>
+          <NavBar
+            left={<View />}
+            center={<BarbellTitle />}
+            right={
+              <TouchableOpacity
+                testID='message-button'
+                onPress={() => router.push('notif/' as Route<string>)}
+              >
+                <FontAwesome5 name='inbox' size={24} color='#CACACA' />
+              </TouchableOpacity>
+            }
+          />
+          <View className='mt-4 px-4'>
+            <Text className='text-4xl font-semibold text-white'>
+              Hello {userData?.name.split(' ')[0]}!
+            </Text>
+          </View>
 
-      <Button
-        onPress={() => router.push('nav/')}
-        className='my-5 flex h-10 items-center rounded-md bg-blue-500'
-        aria-label='Go to nav'
-      >
-        <Text className='text-white'>Nav</Text>
-      </Button>
+          <View className='mt-6'>
```

Comment:
Can we put these in a ScrollView? We are going to have multiple components on the Dashboard

Reply:
For now Im going to leave it as a View because I am getting a nested scrollview error since the feed component is a scroll view.